doc/reference/provided_metrics.md

```diff
@@ -154,6 +158,28 @@ The following internal metrics are provided:
   - Number of active warnings
 ```
 
+(api-rates-metrics)=
+## API rates metrics
+
+The API rates metrics include the `lxd_api_requests_completed_total` and `lxd_api_requests_ongoing`. These metrics can be consumed by an observability tool deployed externally (either Canonical COS or third parties) to help identify failures or overload on a LXD server. Thresholds for these metrics' values can be set on these tools to trigger alarms and take programatic actions.
+These metrics consider all endpoints in the [LXD REST API](api.md), plus the internal endpoints. They also include a label `entity_type` based on the main entity type that the endpoint is operating on. This label can have one of the following values:
+- `instance`
+- `image`, for endpoints that operate on images or image aliases.
+- `storage_pool`, for endpoints that operate on storage pools, storage volumes, storage backups or storage buckets.
+- `network`, for endpoits that operate on networks, network zones or network ACLs.
+- `operation`
+- `project`
+- `warning`
+- `profile`
+- `identity`, for endpoints that related to authorization.
+- `node`, for endpoints related to clustering.
+- `server`, for internal endpoints and any remaining endpoints that don't operate on any of the entity types above.
+
+`lxd_api_requests_ongoing` consists of the number of requests that are not yet completed by the time the metrics are queried. A request is considered completed when the response is returned to the client and any asynchronous operations spawned by that request are done. `lxd_api_requests_completed_total` consists of the number of completed requests. This metric includes an additional label named `result` based on the outcome of the request. The label can assume on of the following values:
+- `error_server`, for errors on the server side, this includes responses with HTTP status codes from 500 to 599. Any failed asynchronous operations also fall into this category.
```

Comment:
```suggestion
- `error_server`, for errors on the server side, including responses with HTTP status codes from 500 to 599. Any failed asynchronous operations also fall into this category.
```

Reply:
For this list, having a full stop at the end of all items makes sense since you need it for this item.

Reply:
Got it, thanks for the insight!